WORKSPACE

```diff
@@ -68,6 +78,15 @@ container_pull(
     digest = "sha256:3b77ee8bfa6a2513fb6343cfad0dd6fd6ddd67d0632908c3a5fb9b57dd68ec1b",
 )
 
+container_pull(
+    name = "bazel-docker-gcloud",
+    registry = "gcr.io",
+    repository = "asci-toolchain/nosla-ubuntu16_04-bazel-docker-gcloud",
```

Comment:
What is this repository?

Reply:
The gcr.io/asci-toolchain is a Google-managed repository, apparently. https://cloud.google.com/vpc-service-controls/docs/supported-products

I got the repo address from https://github.com/GoogleCloudPlatform/container-definitions/tree/80de5376874035319403400fa253b2785c31f3e2/ubuntu1604_bazel_docker_gcloud